packages/manager/modules/pci/src/components/users/download-rclone/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import component from './download-rclone.component';
+
+const moduleName = 'ovhManagerPciComponentsHdsOption';
```

Comment:
You have to rename it :) :
```suggestion
const moduleName = 'ovhManagerPciComponentsUsersDownloadRCloneComponent';
```

Reply:
Okay Mohammed